viz/README.md

```diff
@@ -0,0 +1,21 @@
+## Run
+
+Make sure yarn is installed
+
+    (mac): (mealscount-backend/viz)$ brew install yarn
+    (linux): (mealscount-backend/viz)$ apt install yarn
+
+Go to project root and generate data in viz/src/data
```

Comment:
super minor. 

update commit and add `viz/src/data` folder

Update this for the location and file name.
`viz/src/data/data.json`

Reply:
Looks good.  

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/6610406/57589727-188c0480-74db-11e9-8cf5-9412f8c3292f.png">